repository/s.json

```diff
@@ -4997,6 +4997,19 @@
 				}
 			]
 		},
+		{
+			"name": "Sublitesse Color Scheme",
+			"description": "A calm, green-ish tint Color Scheme. Based on Ant Fu's VSCode Theme",
```

Comment:
If you put a description on your GitHub repo, you can remove it here.
Also, you can remove the “issues” here since it’s just the default url for a GitHub repo.

Reply:
Thank you! I've added the changes. I also fixed the supported version (https://github.com/wbond/package_control_channel/pull/8738/commits/03051ed6148db9ddb666a52480457e86b0fc983e)